### PR TITLE
Yogsbox now has a combined Medsci Surgical Center

### PR DESCRIPTION
**Pull request by Nickvr628**
image
*ignore the PDA and pen and stuff, those were leftover items from me testing it*

I never really liked that the surgery room was this annex in the back corner of medbay, and that robotics just had a small closet to do it in, so I combined made a combined medsci surgery center located between the two departments. This encourages science and medical doctors to perform the dozen-plus beneficial surgeries available and gives the crew a place to go and ask for them. It also makes yogsbox more unique than the TG version, which is always a good thing. The amount of surgical tools and related items are unchanged from before so balance/weapon availability is mostly unchanged.

Anyone with surgery, science, or robotics access can enter through the waiting room, only robotisists can use the direct entrance from their department. There is also surgical storage room that holds the blood bag and prosthetic boxes, as well as the organ smartfridge.

Paramedics and morgue were displaced by this change, and now exist where the old surgical suite and useless observation chamber were before. This means the chef has to be let into medbay to get through the medbay entrance to access the corpses but it is just a minor inconvenience. 

image
EDIT: The erroneous floor tile in the image has been fixed.

The mech bay also was shifted downward and opens into science instead of the main hallway.

TL;DR: The station now has a surgical center that encourages medical and science staff to perform surgeries on the crew, instead of surgery being an afterthought in the station layout.

:cl:  
rscadd: Yogsbox now has a combined medsci surgical center
/:cl:

### CODE REVIEW

**Comment by Firewolf34** [comment]
This is dope, did this have anything to do with the discussions in OOC last night? People were discussing how Yogbox needed to be more unique and how medical needed improvements.

Only issue I see with this is the Mechbay. There's no way for people to bring their mech in to charge without having Science access, this could be a problem. Before you could just leave the garage open for them. Do you think there's any way you could provide a path for mechs to enter the Mechbay from Central Primary, or no? Seems kinda infeasible given the layout, though...

**Comment by Firewolf34** [comment]
Oh and I suggest moving those two computers in the paramed area to the opposite wall. Then putting a door in, so the para's can exit into Central quickly.

**Comment by Nickvr628** [comment]
Yeah it was influenced by that and some chats with Kojuh today, the mech bay issue is unfortunate but the robotisist has to give it science access to have it walk out the front door of science anyway, so it shouldn't be a huge issue.

I will think about giving the paramedics a door to the outside hallway, it would be easy to swap the window with a door. If others could chime in on opinions of that it would be appreciated.

**Comment by Firewolf34** [comment]
I think you should leave the window, it would look nice with the airlock beneath the window, in my opinion. Up to you, though, at the end of the day.

I think the mechbay issue is a fair trade off given how much of a boon this will be to medbay in general. I think it's worth it.

Also it seems you don't have a beaker of styptic on top of your surplus prosthetics crate in surgery storage.

**Comment by Nickvr628** [comment]
It is inside the crate

**Comment by Firewolf34** [comment]
> It is inside the crate

You truly thought of everything. Good work!

**Comment by Chuggachar** [comment]
I like it but it probably needs some tweaks, there shouldn't only be a one door barrier between randoms and surgery tools, unsure about such easy access from breaking in through the hall either, maybe lock the table and tools behind a shutter or something that requires med/robo access? Paramedics definitely need access to the hall, also seems weird to me that there is so many windows into the area, especially storage.

**Comment by Nickvr628** [comment]
I dislike small rooms without windows, feels too cramped and they are not aesthetically pleasing. For now I am not giving paramedics access to the surgery center because that is not really their role, maybe in a future PR that can be changed. I have a solution to make it a bit more secure that I am working on now.

**Comment by Nickvr628** [comment]
image

Updated the surgical suites with a glass shield as extra greytide prevention and to prevent someone bumping into the surgeon by accident, and changed the entrance to make it more visually appealing

**Comment by Chuggachar** [comment]
Didn't even think of windoors, looks good.

**Comment by Ratvar** [comment]
I am still not sure removing botany, chef, chaplain and robotic access to bodies, thus preventing them from podcloning, cooking, burying and borging, is a good idea in attempt to promote surgery.
What about detectives and security, can those open morgue maint doors?

**Comment by OppressedCactus** [comment]
Something to remember is that it is impossible to change a layout without reorganizing things, no matter what he did someone would complain about some element of the change. There are drawbacks to this change but I think we should testmerge it and see if they are actually as bad as some think.

Personally, I like it.

**Comment by Nickvr628** [comment]
https://cdn.discordapp.com/attachments/134722107190935552/614638413085147302/image0.png

Fuck. Will reopen for the mark-2 KTL Boogaloo revision

**Comment by Firewolf34** [comment]
Aww crap. I was lookin' forward to this.
